src/core/cff_font.js

```diff
@@ -74,7 +74,10 @@ class CFFFont {
       return charCodeToGlyphId;
     }
 
-    const encoding = cff.encoding ? cff.encoding.encoding : null;
+    let encoding = cff.encoding ? cff.encoding.encoding : null;
+    if (properties.isStandardFont) {
```

Comment:
Wait, shouldn't this actually be `properties.isInternalFont` now given the change to this parameter?
This might be related to the movement in the reference tests!?

Reply:
I don't think this should matter, since standard fonts should use the base encoding, but we can change it for now. 

As for the ref test changes, I too was confused by that. It turns out it was because I added several of the standard fonts to `getStdFontMap` that were not in there before. This caused `isStandardFont` to be `true` in https://github.com/mozilla/pdf.js/blob/8b4acb4e3683e544f3b4840b3b9a8bfa70f36102/src/core/fonts.js#L967 and subsequently changed the value of `remeasure`. It doesn't seem to change much on the bots, but on macos the fonts seem to look better now for unix01.